developer/clean_dist_check.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```

Comment:
not sure, isn't much less code if we do in the `tox.ini`
```
allowlist_externals = unlink
commands_pre = 
    unlink {toxinidir}/dist/*.whl`
    unlink {toxinidir}/dist/*.tar.gz
```


Also it is a cleanup for `tox -e build` right?

Reply:
yes you are right

Reply:
```
allowlist_externals = bash
commands_pre =
    bash -c "if [ -e {toxinidir}/dist/*tar.gz ]; then unlink {toxinidir}/dist/*.whl; fi"
    bash -c "if [ -e {toxinidir}/dist/*tar.gz ]; then unlink {toxinidir}/dist/*.tar.gz; fi"

```
I don't mind to have a script folder for development tools and to have it in Python, but in this case my impression is that it is not so complicated to do it in bash to include it in the tox.

Reply:
Yes would be fine for me. I check the code.